tools/tfdoc.py

```diff
@@ -373,24 +398,34 @@ def get_readme(readme_path):
     raise SystemExit(f'Error opening README {readme_path}: {e}')
 
 
-def replace_doc(readme_path, doc, readme=None):
+def render_doc(readme, doc):
   'Replace document in module\'s README.md file.'
-  readme = readme or get_readme(readme_path)
   result = get_doc(readme)
-  if not result:
-    raise SystemExit(f'Mark not found in README {readme_path}')
```

Comment:
I would keep this exit message

Reply:
The meaning changed a bit from the previous version. This function now only ties to render the document (vars, outputs, etc) or leaves it as is. After this is done we do the TOC.

I don't want this function to raise, otherwise we would never render the TOC.

Reply:
But then if we don't have a block we never notice...

Reply:
should we fail in that case? I mean, is the mark mandatory in all READMEs?

Reply:
yep, otherwise there's no point running tfdoc :)